allow mobs to walk on stone

SwissalpS opened the pull request:
-- mobs and beowolf don't play nicely together when stone is made
-- dangerous by beowolf, we counteract that here to make mobs not fear
-- stone and become more challenging again

Note: this can only be merged when mobs_redo has made ``mobs.is_node_dangerous(mob_obj, nodename)`` public
as in: https://notabug.org/TenPlus1/mobs_redo/pulls/104

SwissalpS committed:
allow mobs to walk on stone

S-S-X commented on mobs.lua:
```diff
@@ -0,0 +1,9 @@
+-- mobs and beowolf don't play nicely together when stone is made
+-- dangerous by beowolf, we counteract that here to make mobs not fear
```
```suggestion
-- mobs and beowulf don't play nicely together when stone is made
-- dangerous by beowulf, we counteract that here to make mobs not fear
```
Source: https://en.wikipedia.org/wiki/Beowulf

SwissalpS replied:
thanks :)

SwissalpS committed:
add version check and update to newest version

also adjusted varname to what author is using
added mobs to optional dependencies for perfectness sake
and fixed beowulf typo
